webapp/components/search_results.jsx

```diff
@@ -141,14 +141,14 @@ export default class SearchResults extends React.Component {
                     <h4>
                         <FormattedMessage
                             id='search_results.noResults'
-                            defaultMessage='NO RESULTS'
+                            defaultMessage='No results found. Try again?'
```

Comment:
Good topic to bring up, 

In my mind, we have "fast, obvious, forgiving" guidelines, we don't have a guidelines around being verbose, or how to make trade-offs between verbose-ness and other priorities. 

E.g. Our LDAP settings have a lot of help text that isn't needed by LDAP experts, but when we add it our support questions are significantly reduced. 

Propose we figure out a guideline so we can be consistent in the trade-off between helping the least savvy user and having more text than experts need. 

Until we have a standard, I think stay focused on "fast, obvious, forgiving." 

For this text change specifically, our goal is to have the user read the text beyond "No results" and a question seemed the best way to do it (versus bold text).